docs/_config.yml

```diff
@@ -1,5 +1,5 @@
 title: "CanoE-COACH"
-theme: minima
+theme: jekyll-theme-cayman
```

Comment:
this nicer?

Reply:
oh I forgot to change it back, should be fixed now